### DISCUSSION TITLE
Items counter position and scrollbar for thumbnails

### DISCUSSION THREAD

**Discussion by iGer98** [igordanchenko/yet-another-react-lightbox#283] [Q&A] [answered]
Is it possible to move items counter right side over a thumbnail block? I did not found an answer for it and it looks there is no a good solution for it.

<img width="1485" alt="Screenshot 2024-06-18 at 16 48 16" src="https://github.com/igordanchenko/yet-another-react-lightbox/assets/163126598/b87a3fd5-d82d-4466-a763-e35430459d72">
--------------------------------------------------------

Can we somehow add ability to scroll thumbnails?

<img width="1512" alt="Screenshot 2024-06-18 at 16 48 01" src="https://github.com/igordanchenko/yet-another-react-lightbox/assets/163126598/0eb804f1-ee6f-40ae-b0f1-3f13e4a77676">

**Comment by igordanchenko** [comment, marked as answer]
> Is it possible to move items counter right side over a thumbnail block? I did not found an answer for it and it looks there is no a good solution for it.

I'm afraid there is no such option in the Counter plugin, but you can easily implement your own plugin for this use case.

```jsx
// Counter.tsx

import {
  PluginProps,
  PLUGIN_THUMBNAILS,
  createModule,
  ComponentProps,
  useLightboxState,
} from "yet-another-react-lightbox";

function CounterComponent({ children }: ComponentProps) {
  const { slides, currentIndex } = useLightboxState();

  return (
    <>
      {children}

      <div
        style={{
          color: "white",
          position: "absolute",
          right: 16,
          top: 16,
        }}
      >
        {currentIndex + 1} / {slides.length}
      </div>
    </>
  );
}

export default function Counter({ addParent }: PluginProps) {
  addParent(PLUGIN_THUMBNAILS, createModule("counter", CounterComponent));
}
```

> Can we somehow add ability to scroll thumbnails?

No, I'm afraid not.

https://github.com/igordanchenko/yet-another-react-lightbox/discussions/193#discussioncomment-7139493

**Reply by iGer98** [reply]
Thank you so much for a such quick response :)
